core/modules/ccontrol/UserQueryFactory.cc

```diff
@@ -101,19 +103,19 @@ UserQueryFactory::newUserQuery(std::string const& query,
         qs->setQuery(query);
     } catch (...) {
         errorExtra = "Unknown failure occured setting up QuerySession (query is invalid).";
-        LOGF_ERROR(errorExtra);
+        LOGF(_log, LOG_LVL_ERROR, errorExtra);
         sessionValid = false;
     }
     if(!qs->getError().empty()) {
-        LOGF_ERROR("Invalid query: %s" % qs->getError());
+        LOGF(_log, LOG_LVL_ERROR, "Invalid query: %s" % qs->getError());
         sessionValid = false;
     }
     if(!qs->getError().empty()) {
-        LOGF_INFO("Invalid query: %s" % qs->getError());
+        LOGF(_log, LOG_LVL_INFO, "Invalid query: %s" % qs->getError());
         sessionValid = false;
     }
     if(!qs->getError().empty()) {
-        LOGF_INFO("Invalid query: %s" % qs->getError());
+        LOGF(_log, LOG_LVL_INFO, "Invalid query: %s" % qs->getError());
         sessionValid = false;
     }
```

Comment:
Does the code above repeat the same `if () {...}` three times (once with ERROR and twice with INFO)?